samoa-api/src/main/java/org/apache/samoa/evaluation/EvaluatorProcessor.java

```diff
@@ -179,6 +211,28 @@ private void addMeasurement() {
     }
   }
 
+  /**
+   * This method is used to create one line of a text file containing predictions and votes (for classification only).
+   * In case, this is the first line a header line is also added
+   */
+  private void addVote() {
+    List<Vote> votes = new Vector<>();
+    Collections.addAll(votes, evaluator.getPredictionVotes());
+    Vote[] finalVotes = votes.toArray(new Vote[votes.size()]);
+    learningCurve.setVote(finalVotes);
+    logger.debug("evaluator id = {}", this.id);
+
```

Comment:
The important role of Vote-related methods in LearningCurve is to ensure that Vote objects are placed every time in the same sequence (which is done by setVote()). This is important to ensure that individual columns are printed in every line of prediction file in the same sequence. Hence, we suggest to keep the current solution, which uses learningCurve. The addVoteName() method is consistent with addMeasurementName() used for the same reason for measurements.